### ISSUE TITLE
Crashing when deleting objects using the edit window

### ISSUE THREAD

**Issue by guerreiro-osorio** [veusz/veusz#247]
Veusz 2.2.2 on Linux Mint 18.3

Is crashing whenever I delete a selection of objects - if the objects are labels, the app just terminate, for other objects, it raises the error tracker
[CurvaNormalPadrao.vsz.txt](https://github.com/veusz/veusz/files/1927818/CurvaNormalPadrao.vsz.txt)

.

**Comment by jeremysanders** [comment]
Unfortunately I can't reproduce this on Ubuntu linux. Is this the binary download, the distribution package or did you build this yourself?

**Comment by guerreiro-osorio** [comment]
Dear Jeremy,

I was using the PPA version, with Mint 18.3, but then I found out that something odd happened with the machine I was using, as Libre Office was also crashing and freezing. I have Veusz in other machine with the same OS and software, and it was working fine, so instead of looking for the problem, I just reinstalled everything and now I am using the PPA version again, and Veusz is working fine, as usual.

Thank you for your concern and for Veusz, I have been using it for 3-4 years, and is definitely my favorite plotting tool.

Best,

Rafael Osorio


De: Jeremy Sanders [mailto:notifications@github.com]
Enviada em: sábado, 5 de maio de 2018 08:40
Para: veusz/veusz <veusz@noreply.github.com>
Cc: Rafael Guerreiro Osório <rafael.osorio@ipea.gov.br>; Author <author@noreply.github.com>
Assunto: Re: [veusz/veusz] Crashing when deleting objects using the edit window (#247)


Unfortunately I can't reproduce this on Ubuntu linux. Is this the binary download, the distribution package or did you build this yourself?
